build.sbt

```diff
@@ -1,10 +1,36 @@
 import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import scoverage.ScoverageKeys
 
 val appName = "minor-entity-identification-frontend"
 
 val silencerVersion = "1.7.3"
 
+lazy val scoverageSettings = {
+
+  val exclusionList: List[String] = List(
+    "<empty>",
+    ".*Routes.*",
+    ".*Reverse.*",
+    "app.*",
+    "prod.*",
+    "config.*",
+    "com.kenshoo.play.metrics.*",
+    "testOnlyDoNotUseInAppConf.*",
+    "uk.gov.hmrc.minorentityidentificationfrontend.featureswitch.api.*",
+    "uk.gov.hmrc.minorentityidentificationfrontend.featureswitch.frontend.*",
+    "uk.gov.hmrc.minorentityidentificationfrontend.testonly.*",
+    "uk.gov.hmrc.minorentityidentificationfrontend.views.html.*"
+  )
+
+  Seq(
+    ScoverageKeys.coverageExcludedPackages := exclusionList.mkString(";"),
+    ScoverageKeys.coverageMinimum := 90,
```

Comment:
For some reason "coverageMinimum" is shown as being deprecated in minor-entity. We are recommended to use "coverageMinimumStmtTotal" instead.